### PR TITLE
Added in release notes for 1.5.5

### PR DESCRIPTION
**Pull request by michaelgsharp**
Here are the release notes for 1.5.5. #5634 needs to go in before these notes do.

### CODE REVIEW

File: docs/release-notes/1.5.5/release-1.5.5.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+## **Build / Test updates**
+- Fixing official build by adding homebrew bug workaround ([#5596](https://github.com/dotnet/machinelearning/pull/5596))
+- Nuget.config url fix for roslyn compilers ([#5584](https://github.com/dotnet/machinelearning/pull/5584))
+- Add SymSgdNative reference to AutoML.Tests.csproj ([#5559](https://github.com/dotnet/machinelearning/pull/5559))
+
+
+## **Documentation Updates**
+- Updated documentation for the correct version of CUDA for TensorFlow. ([#5635](https://github.com/dotnet/machinelearning/pull/5635))
+- Updates cocumentation for an issue with brew and installing libomp. ([#5635](https://github.com/dotnet/machinelearning/pull/5635))
```
Comment (ericstj) [inline docs/release-notes/1.5.5/release-1.5.5.md:33]:
```suggestion
- Updates documentation for an issue with brew and installing libomp. ([#5635](https://github.com/dotnet/machinelearning/pull/5635))
```

File: docs/release-notes/1.5.5/release-1.5.5.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+- Updated an ONNX url to the correct url.  ([#5635](https://github.com/dotnet/machinelearning/pull/5635))
+- Added in note in the documentation that the PredictionEngine is not thread safe. ([#5583](https://github.com/dotnet/machinelearning/pull/5583))
```
Comment (ericstj) [inline docs/release-notes/1.5.5/release-1.5.5.md:35]:
```suggestion
- Added a note in the documentation that the PredictionEngine is not thread safe. ([#5583](https://github.com/dotnet/machinelearning/pull/5583))
```

**Review by ericstj — APPROVED** [review]

File: docs/release-notes/1.5.5/release-1.5.5.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# [ML.NET](http://dot.net/ml) 1.5.5
+
+## **New Features**
+- **New API allowing confidence parameter to be a double**.([#5623](https://github.com/dotnet/machinelearning/pull/5623))
+. A new API has been added to accept double type for the confidence level. This helps when you need to have high precision then an int will allow for. (**Thank you @esso23**)
```
Comment (tarekgh) [inline docs/release-notes/1.5.5/release-1.5.5.md:5]: did you mean `than` and not then?

File: docs/release-notes/1.5.5/release-1.5.5.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+- **New API to treat TensorFlow output as batched/not-batched** ([#5634](https://github.com/dotnet/machinelearning/pull/5634)) A new API has been added so you can specify if the output from TensorFlow is batched or not.
+
+
+## **Enhancements**
+- Make ColumnInference serializable ([#5611](https://github.com/dotnet/machinelearning/pull/5611))
+
+
+## **Bug Fixes**
+- **AutoML.NET specific fixes**.
+  - Fixed am AutoML aggregate timeout exception ([#5631](https://github.com/dotnet/machinelearning/pull/5631))
```
Comment (tarekgh) [inline docs/release-notes/1.5.5/release-1.5.5.md:16]: what is `am'?

**Review by tarekgh — APPROVED** [review]
added a couple of comments/questions. otherwise LGTM.

File: docs/release-notes/1.5.5/release-1.5.5.md (hunk 3 above)
Comment (michaelgsharp) [reply docs/release-notes/1.5.5/release-1.5.5.md:5]: Yes I did. I always make that mistake, so thanks for catching it.

File: docs/release-notes/1.5.5/release-1.5.5.md (hunk 4 above)
Comment (michaelgsharp) [reply docs/release-notes/1.5.5/release-1.5.5.md:16]: Should be `an`, good catch.

**Pull request merged**